src/handlers/gestures/gestureComposition.ts

```diff
@@ -72,24 +72,19 @@ export class ComposedGesture extends Gesture {
 }
 
 export class SimultaneousGesture extends ComposedGesture {
-  constructor(first: Gesture, second: Gesture) {
-    super(first, second);
-  }
-
   prepare() {
-    const leftSide = this.gestures[0].toGestureArray();
-    const rightSide = this.gestures[1].toGestureArray();
+    const simultaneousArray = this.gestures
+      .map((gesture) => gesture.toGestureArray())
+      .flat()
```

Comment:
`.flatmap()`

Reply:
Done in c6337844ec285e9632eead05b257fc869adfdff4